content/docs/components/badge/theming.mdx

```diff
@@ -33,7 +33,7 @@ import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
 ```jsx live=false
 import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
 
-const boxy = defineStyle({
+const outline = defineStyle({
```

Comment:
Hey! 🙌 Thank you for your contribution 💖 Can we actually just rename the export instead since the badge is actually box-y and this name is pretty well descriptive. 

```js
export const badgeTheme = defineStyleConfig({
  variants: { boxy },
})
```

Reply:
Sounds good, made that change 👍

Reply:
Thanks, made that change 👍